design/one-pager-stacks-versioning.md

```diff
@@ -0,0 +1,248 @@
+# Stack Versioning
+* Owner: Luke Weber (@lukeweber)
+* Reviewers: Crossplane Maintainers
+* Status: Draft, revision 1.0
+
+## Problem
+
+Stacks installer currently relies on installing a particular version of stack from a docker tag. The stack installer in
+ the future will be able to support stack update policy, maintenance windows, warnings on major updates, and more.
+ To build more complex features around stack version management we will need a common way to express versions that is
+ widely understood and standards based, so we will unify on best practices and
+ [semantic version 2.0](https://semver.org/spec/v2.0.0.html) as the Crossplane versioning standard.
+
+#### Didn't docker tags already solve this?
+
+ * I don't want my versions underlying artifacts to change:
+    * Tags are a symbolic reference to an actual sha of an image and they can be updated or deleted
+ * I want to have context from a version:
+    * A sha is hard to compare when debugging
+      * Which is newer? - cb44dd0 or d2fb617?
+      * Is this alpha, beta?
+    * A tag like latest is opaque to the end user and so not incredibly useful:
```

Comment:
Since stacks are in general built from a repo, what are our thoughts on where the gitref of the source comes into play? It seems like it may be useful for whenever someone (probably a developer) is trying to debug a problem by pulling up the source code.